src/components/Contentful/DynamicRichText.vue

```diff
@@ -0,0 +1,41 @@
+<script>
+import KvContentfulImg from '~/@kiva/kv-components/vue/KvContentfulImg';
```

Comment:
We'll have to import all possible components that can be in a rich text field here.

Reply:
You could use async imports, and then they won't be imported until they're used:
```
const KvContentfulImg = () => import('~/@kiva/kv-components/vue/KvContentfulImg');
```